source/trx2junit/trx2junit.csproj

```diff
@@ -5,6 +5,7 @@
         <TargetFrameworks>net6.0;net5.0;netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
         <PackAsTool>true</PackAsTool>
         <Nullable>enable</Nullable>
+        <RollForward>Major</RollForward>
```

Comment:
Cf. https://github.com/dotnet/designs/blob/main/accepted/2019/runtime-binding.md#rollforward

Is this the correct value?